### PR TITLE
Created basic venue model

### LINKED ISSUE #40: Develop an entity object Venue.
- [ ] Create DB entity Venue.

### PR DESCRIPTION
**Pull request by KEDuran**
I created a basic venue model located in this folder path - GitHub\wedding-planner\models. This model will not work until the actual database and data connections are created.

Please note that I did not include the connection configuration in this branch to avoid any merge conflicts.

closes #40

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,13 +20,14 @@
...
-    "nodemon": "^1.18.7",
-    "mysql2": "^1.6.5"
+    "mysql2": "^1.7.0",
+    "nodemon": "^1.18.7"
   },
   "dependencies": {
     "babel-eslint": "^10.0.1",
     "express": "^4.17.1",
     "if-env": "^1.0.4",
+    "mysql": "^2.18.1",
```
Comment (ystamaritq) [inline package.json]: we don't need this in production as we will be using a postgres database.

Comment (ystamaritq) [reply package.json]: let's make sure to delete this  ("mysql": "^2.18.1") dependency I suggest tested in heroku but I think will be fine.

Comment (KEDuran) [reply package.json]: Thank you for the review! I will remove this package

Comment (KEDuran) [reply package.json]: I just removed the package!

**Review by ystamaritq — APPROVED** [review]
great! thanks

**Pull request merged**